packages/site/src/data/react/courses.ts

```diff
@@ -15,19 +15,21 @@ export const courseTags = [
 	"bootstrap",
 	"react context",
 	"hooks",
-	"typescript",
+	"TypeScript",
 	"tailwind",
-	"material UI",
+	"Material UI",
 	"vite",
-	"javascript marathon",
+	"JavaScript marathon",
 	"AWS",
 	"netlify",
 	"continuous integration",
-	"gatsbyJS",
+	"GatsbyJS",
 	"SSG",
 	"component design",
 	"apollo",
-	"faunaDB",
+	"FaunaDB",
+	"sass",
+	"static site generation",
```

Comment:
Should we have SSG and static site generation? I think spelling it out is probably better, but SSG is on line 27 if we don't want both.

Reply:
I think we should go with static site generation and drop SSG